### PR TITLE
Exceptions to status code

### PR DESCRIPTION
**Pull request by soyuka**
Document https://github.com/api-platform/core/pull/797 in configuration

Should this be added to another chapter too?

### CODE REVIEW

File: core/configuration.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ api_platform:
 
     # Enable the Nelmio Api doc integration.
     enable_nelmio_api_doc: false
+
+    # Map exceptions to http status codes
```
Comment (teohhanhui) [inline core/configuration.md]: HTTP

File: core/configuration.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ api_platform:
...
+    exception_to_status:
+        Symfony\Component\Serializer\Exception\ExceptionInterface: 400 # with a status code
+        ApiPlatform\Core\Exception\InvalidArgumentException: 'HTTP_BAD_REQUEST' # or with a constant defined in the 'Symfony\Component\HttpFoundation\Response' class
```
Comment (teohhanhui) [inline core/configuration.md]: This is a misfeature. Sorry that I didn't catch this.

Symfony 3.2 will support using PHP constants in YAML: http://symfony.com/blog/new-in-symfony-3-2-php-constants-in-yaml-files

We should remove this.

Comment (soyuka) [inline core/configuration.md]: https://github.com/api-platform/core/pull/797/files#diff-a01015ec3cfbd89ee246335fd528c816R114 it's done manually there

Comment (teohhanhui) [inline core/configuration.md]: Yes, I mean we should remove it.

**Comment by dunglas** [comment]
Thanks @soyuka

**Pull request merged**